conans/client/cmd/uploader.py

```diff
@@ -84,7 +83,7 @@ def _upload(self, conan_file, conan_ref, force, packages_ids, retry, retry_wait,
         else:  # Or use the default otherwise
             upload_remote = self._registry.default_remote
 
-        if not force:
+        if no_overwrite != "force":
```

Comment:
This comparison is a bit confusing. I know doing it this way there is one parameter less but I think the old code reads better IMO

Reply:
Yeah, I know, but the alternative is passing around 2 mutually exclusive arguments, which is bad interface. I thought changing both for a ``policy``, but wanted to keep changes to a minimum.

Reply:
introduce the policy please, I really hate that kind of arguments